### PR TITLE
Build seeds-packets using github actions and yarnpkg.com

### PR DESCRIPTION
**Pull request by david-huber**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          node-version: 16.14.2
+      - name: Setup .npmrc
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_PUBLISH_TOKEN }}" > .npmrc
+      - name: Install Dependencies
+        run: yarn
+      - name: Build
+        run: yarn build
+      - name: Publish
+        run: npx lerna publish from-package --no-push --yes
```
Comment (david-huber) [inline .github/workflows/publish.yml:28]: ... if main@HEAD contains new packages in it, publish them to npmjs.org. DO NOT push anything to github.

File: .github/workflows/version.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Version
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
```
Comment (david-huber) [inline .github/workflows/version.yml:6]: whenever a PR against main is closed ...

File: .github/workflows/version.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  release:
+    if: github.event.pull_request.merged == true
```
Comment (david-huber) [inline .github/workflows/version.yml:9]: ... and it was actually merged

File: .github/workflows/version.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Setup Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
+      - name: Install Dependencies
+        run: yarn
+      - name: Build
+        run: yarn build
+      - name: Version
+        run: npx lerna version --conventional-commits --yes
```
Comment (david-huber) [inline .github/workflows/version.yml:27]: ... then version it using conventional commits and push the never versions back to github.

File: .nvmrc [hunk 5]
```diff
@@ -1 +1 @@
-12
\ No newline at end of file
+16
```
Comment (david-huber) [inline .nvmrc:1]: A previous PR moved the build version to node v. 16, but this was not updated.

File: package.json [hunk 6]
```diff
@@ -19,9 +19,7 @@
   },
   "scripts": {
     "format:prettier": "prettier-eslint --write --eslint-ignore --print-width=120 --single-quote --no-bracket-spacing",
-    "build": "lerna run build",
-    "release-artifactory": "lerna publish --registry=\"https://sproutsocial.jfrog.io/artifactory/api/npm/npm_local\" --conventional-commits --yes",
-    "release-npmjs": "lerna publish from-package --no-push --yes"
+    "build": "lerna run build"
```
Comment (david-huber) [inline package.json:22]: removing all release/versioning scripts, as they should not be used outside our workflows.

**Comment by lmosele** [comment]
Thanks for the notes! helpful for future reference 👍🏻

**Pull request merged**